src/audio/src.c

```diff
@@ -975,3 +977,5 @@ void sys_comp_src_init(void)
 {
 	comp_register(&comp_src);
 }
+
+DECLARE_COMPONENT(sys_comp_src_init);
```

Comment:
Is there any specific reason why the DECLARE is put after the function definition, not right after the declaration? If so, please document for other components, otherwise might be more readable to keep the DECLARation along with the declaration.

Reply:
I guess the reason is that it's recommended by our guidelines: https://www.kernel.org/doc/html/v4.11/process/coding-style.html#functions

Also EXPORT_COMPONENT may be better name

Reply:
Yes, it should be just after definition like jajanusz descibed